Xamarin.Forms.Maps/Pin.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Xamarin.Forms.Maps
 {
-	public class Pin : BindableObject
+	public class Pin : Element
```

Comment:
why did you had to change this ?

Reply:
Per my understanding, `Pin` has to be an `Element` in order to be the root of `DataTemplate`. Please correct me if I'm wrong, I think the technical way to describe is, it's part of the logical tree but not the visual tree. I think same pattern exists in ListView? (the cells)

Reply:
👍